tests/integration/targets/omem_device_network_services/tests/invalid.yaml

```
@@ -0,0 +1,162 @@
+# Test code for the Dell OpenManage modules
+# Copyright (C) 2025 Dell Inc. or its subsidiaries. All Rights Reserved.
+
+# GNU General Public License v3.0+ (see COPYING or
+# https://www.gnu.org/licenses/gpl-3.0.txt)
+---
+- name: Test that we have an OME host, OME username and OME password
+  fail:
```

```suggestion
  ansible.builtin.fail:
```
Add FQDN for all subtask across testcases file.

---

done